analyses/equivalency_cliques.py

```diff
@@ -0,0 +1,104 @@
+"""Equivalency cliques analyses
+
+Prerequisites
+  - Files: Need to be pre-downloaded in data/.
+
+Resources
+  - GitHub issue: https://github.com/monarch-initiative/omim/issues/45
+
+Areas for improvement
+  - Dynamic term_id list: Instead of a static list, could do something dynamic, like figuring out which of these terms
```

Comment:
This would be good, else we dont know for the next release which equivalent cliques will be active :)

Reply:
OK, I will go ahead and do that then. Thanks for reviewing those notes.

Reply:
I think a trivial check would be to check if any one subject is shared by more than one object, and vice versa?

Reply:
probably this wont catch cycles (A->B->C->D), but should catch most of them.

Reply:
@matentzn I"m going to merge this, but the question above still stands. Still wondering why we aren't utilizing Entrez and Ensembl gene IDs.

Reply:
Open a new issue about this, and lets make sure that we add the answer (will be provided by Chris) to our ingest docs.

Reply:
Done! #66